modules/references/client/components/CreateReference.component.js

```diff
@@ -133,11 +150,11 @@ export default function CreateReference({ userFrom, userTo }) {
           {tabs[1]}
         </Tab>
       </Tabs>
-      {/* <!-- Navigation for big screens -->*/}
+      {/* <!-- Navigation -->*/}
```

Comment:
Fine to remove `<!--` since this is a JS comment now.

Actually, fine to remove altogether since it doesn't tell much anymore. :-)

Reply:
:ok_hand: a15884f55282bf4570726f50d957a9712b8d53d8